generator/src/lib.rs

```diff
@@ -2138,12 +2144,16 @@ fn derive_getters_and_setters(
                 });
             } else if is_static_array(field) {
                 assert_eq!(field.size, None);
-                return Some(quote! {
+                let setter = struct_.is_return.not().then(|| quote! {
                     #deprecated
                     #[inline]
                     pub fn #param_ident_short(mut self, #param_ident_short: &CStr) -> core::result::Result<Self, CStrTooLargeForStaticArray> {
                         write_c_str_slice_with_nul(&mut self.#param_ident, #param_ident_short).map(|()| self)
                     }
+                 });
```

Comment:
formatting issue ?

Reply:
This bunch of code is so large that `rustfmt` refuses to deal with it. I don't like fixing it up (incorrectly) manually either, we should figure out how to split it up into formattable functions.